aws_sagemaker_studio/sagemaker_experiments/track-airflow-workflow/track-an-airflow-workflow.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>Consider adding a comment explaining we're only doing this cleanup because this is an example, you wouldn't actually want to do this if you were really using this tool.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>